packages/components/src/templates/next/components/native/Table/Table.tsx

```diff
@@ -85,66 +87,70 @@ const Table = ({ attrs: { caption }, content }: TableProps) => {
   }, [])
 
   return (
-    <div className="overflow-x-auto [&:not(:first-child)]:mt-6">
-      <table
-        className="relative w-full border-separate border-spacing-0"
-        ref={tableRef}
-      >
-        <caption className="mb-4 caption-top text-left">
-          <BaseParagraph
-            content={caption}
-            className="prose-label-md-regular sticky left-0 table-header-group text-balance text-base-content-subtle"
-          />
-        </caption>
-        <tbody>
-          {content.map((row, index) => {
-            const TableCellTag =
-              row.content[0]?.type === "tableHeader" ? "th" : "td"
+    <div className="flex flex-col gap-4">
+      <BaseParagraph
+        id={tableDescriptionId}
+        content={caption}
+        className="prose-label-md-regular text-base-content-subtle [&:not(:last-child)]:mb-0"
```

Comment:
I think it will always not be the last child?

```suggestion
        className="prose-label-md-regular text-base-content-subtle mb-0"
```

Reply:
no choice. your `BaseParagraph` component has the exact same rule but adds `mb-6` 😆 . I need to override

Reply:
LOL okok damn